leading-teams/working-with-clients.html.md

```diff
@@ -21,13 +21,20 @@ changes that will improve the quality or stability of the product. When this is
 to discuss these improvements with the client and explain why they are important so that they can
 make the right decision.
 
-Finally, a note on making decisions: as a project manager, you will sometimes call the shots on
+Finally, a note on making decisions: as a team lead, you will sometimes call the shots on
 technical (or even business) matters that the client delegates to you. For instance, you may find
 yourself making build-vs-buy decisions, or deciding whether to use a new framework for that feature
 or rely on a more boring, but proven, technology. You should listen and address any feedback from
 the team, but at the end of the day, you must go with the option that _you_ feel is right, because
 it's your job to make the right decisions based on your personal experience. To be able to do this,
 you need to foster the kind of culture where people can [disagree and commit][disagree-and-commit].
 
+As part of this, a team lead will always make sure we are fulfilling both our clients' expectations
+and our team members needs. This is accomplished by:
```

Comment:
You need an apostrophe after members, or you can write "the needs of our team members".